src/equipment_inventory/templates/odm2/organization-details.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'equipment_inventory/details.html' %}
+{% load static %}
+
+{% block detail-title %}
+    <h3>{{ page_title }}</h3>
+{% endblock %}
+
+{% block styles %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'odm2/css/site-detail.css' %}">
+{% endblock %}
+
+{% block header_buttons_toolbar %}
+    <div class="buttons-toolbar row">
+        <div class="col-12 text-right">
+            <a href="{% url 'admin:odm2_organization_change' object.pk %}" id="btn-edit-sampling_feature"
+               class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect mb-2 full-width-sm">
+                <i class="material-icons center-icon">mode_edit</i> Edit Organization
+            </a>
+
+            <a href="{% url 'admin:odm2_organization_delete' object.pk %}"
+               class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect delete mb-2 full-width-sm">
+                <i class="material-icons center-icon">delete_forever</i> Delete
+            </a>
+        </div>
+    </div>
+{% endblock %}
+
+{% block detail_main_section %}
+
+        <div class="col-12">
+            <h5>Organization</h5>
+        </div>
+        <div class="col-12">
+            <table class="mdl-data-table mdl-js-data-table mdl-data-table-no-hover mdl-shadow--2dp">
+                <tr>
+                    <td class="mdl-data-table__cell--non-numeric"><strong>Organization Code</strong></td>
+                    <td class="mdl-data-table__cell--non-numeric">{{ object.organization_code|default:'-' }}</td>
+                </tr>
+                <tr>
+                    <td class="mdl-data-table__cell--non-numeric"><strong>Organization Name</strong></td>
+                    <td class="mdl-data-table__cell--non-numeric">{{ object.organization_name|default:'-' }}</td>
+                </tr>
+                <tr>
+                    <td class="mdl-data-table__cell--non-numeric"><strong>Organization Description</strong></td>
+                    <td class="mdl-data-table__cell--non-numeric">{{ object.organization_description|default:'-' }}</td>
+                </tr>
+                <tr>
+                    <td class="mdl-data-table__cell--non-numeric"><strong>Organization Type</strong></td>
+                    <td class="mdl-data-table__cell--non-numeric">{{ object.organization_type|default:'-' }}</td>
+                </tr>
+                <tr>
+                    <td class="mdl-data-table__cell--non-numeric"><strong>Organization Website</strong></td>
+                    <td class="mdl-data-table__cell--non-numeric"><a target="_blank"
+                           href="{% if "http" not in object.model_link %}
```

Comment:
We should make a template tag/filter that fixes URLs since we'll need this functionality in a bunch of other templates. A template tag would be much cleaner too.